projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,113 @@
+HARBOR_NGINX_IMAGE_COMPONENT=harbor/harbor-nginx
+HARBOR_JOBSERVICE_IMAGE_COMPONENT=harbor/harbor-jobservice
+HARBOR_REGISTRY_IMAGE_COMPONENT=harbor/harbor-registry
+HARBOR_REGISTRYCTL_IMAGE_COMPONENT=harbor/harbor-registryctl
+HARBOR_REDIS_IMAGE_COMPONENT=harbor/harbor-redis
+HARBOR_EXPORTER_IMAGE_COMPONENT=harbor/harbor-exporter
+
+IMAGE_NAMES=harbor-db harbor-portal harbor-core harbor-log harbor-nginx harbor-jobservice harbor-registry harbor-registryctl harbor-redis harbor-exporter
+
+BINARY_TARGET_FILES=harbor-core harbor-jobservice harbor-registryctl harbor-migrate-patch harbor-migrate harbor-exporter
+SOURCE_PATTERNS=./core ./jobservice ./registryctl ./cmd/migrate-patch ./cmd/standalone-db-migrator ./cmd/exporter
+
+
+PRECOMPILE_PREPARE=harbor_prepare
+
+harbor_base_namespace=goharbor
+IMAGE_BUILD_ARGS=harbor_base_namespace
+
+EXTRA_GOBUILD_FLAGS=-mod vendor
+
+FETCH_BINARIES_TARGETS=eksa/distribution/distribution
+
+include $(BASE_DIRECTORY)/Common.mk
+
+$(call IMAGE_TARGETS_FOR_NAME, harbor-db): harbor-db-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-portal): harbor-portal-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-core): harbor-core-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-log): harbor-log-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-nginx): harbor-nginx-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-jobservice): harbor-jobservice-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-registry): harbor-registry-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-registryctl): harbor-registryctl-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-redis): harbor-redis-useradd/images/export
+$(call IMAGE_TARGETS_FOR_NAME, harbor-exporter): harbor-exporter-useradd/images/export
+
+IMAGE_USERADD_USER_ID=10000
+IMAGE_USERADD_USER_NAME=harbor
+
+harbor-db-useradd/images/export: IMAGE_USERADD_USER_NAME=postgres IMAGE_USERADD_USER_ID=999
+harbor-db-useradd/images/export: scripts_prepare
+harbor-portal-useradd/images/export: IMAGE_USERADD_USER_NAME=nginx
+harbor-portal-useradd/images/export: scripts_prepare
+harbor-core-useradd/images/export: scripts_prepare core_config_prepare
+harbor-log-useradd/images/export: IMAGE_USERADD_USER_NAME=syslog
+harbor-log-useradd/images/export: scripts_prepare
+harbor-nginx-useradd/images/export: IMAGE_USERADD_USER_NAME=nginx
+harbor-nginx-useradd/images/export: scripts_prepare
+harbor-jobservice-useradd/images/export: scripts_prepare
+harbor-registry-useradd/images/export: scripts_prepare
+harbor-registryctl-useradd/images/export: scripts_prepare
+harbor-redis-useradd/images/export: IMAGE_USERADD_USER_NAME=redis
+harbor-redis-useradd/images/export: scripts_prepare
+harbor-exporter-useradd/images/export: scripts_prepare
+
+harbor_prepare: version_prepare swagger_prepare
+
+version_prepare:
+	@echo "$$VERSIONS_FOR_PREPARE" > $(PROJECT_ROOT)/$(REPO)/make/photon/prepare/versions
+
+swagger_prepare:
```

Comment:
I ran this locally, looks like it creates files under `src/server/v2.0/models` so can we make this a real target instead? Something like `$(REPO)/src/server/v2.0/models:....`